net/acme/files/run.sh

```diff
@@ -129,6 +129,12 @@ pre_checks()
 
 post_checks()
 {
+    if [ -f "$STATE_DIR/post-acme.user" ]; then
```

Comment:
I think it is better to have the script locations be uci config variables - someone might want to have different ones for different instances, no?

Reply:
I've added this functionality to be the part of the PR too. Script locations can now be defined as configuration variables.